py/desispec/coaddition.py

```diff
@@ -446,6 +446,58 @@ def coadd_fibermap(fibermap, onetile=False):
 
     return tfmap, exp_fibermap
 
+def _mask_cosmics(wave, flux, ivar, mask, subset, ivarjj_masked,
+                  tid=None, cosmics_nsig=None):
+    # interpolate over bad measurements
+    # to be able to compute gradient next
+    # to a bad pixel and identify outlier
+    # many cosmics residuals are on edge
+    # of cosmic ray trace, and so can be
+    # next to a masked flux bin
```

Comment:
Please add a docstring for this new function explaining the inputs (ivarjj_masked?) and which ones are modified in-place.  For testing, I had to add a missing `log = get_logger()`, which implies that the latest version of this code has not been fully tested with cases that actually mask cosmics; I was using
```
desi_coadd_spectra \
    -i $CFS/desi/spectro/redux/kibo/healpix/main/bright/100/10032/spectra-main-bright-10032.fits.gz \
    -o coadd-main-bright-10032.fits
```

Reply:
Will do. Yes mask_cosmics was somewhat of temporary solution, when I hoping to quickly fix the coadd_cameras(), but I still couldn't do it yet. 
I.e. still puzzled by why https://github.com/desihub/desispec/blob/4eae355fbae93e5a138c6ca91074d961c2558286/py/desispec/test/test_coadd.py#L870
this test supposed to return fluxes of 0.5 given the original spectra of 1.